backend/plugins/pluginservice.go

```diff
@@ -106,6 +108,14 @@ func (s *Service) AddHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// -1
+	initPluginsTempDir.Do(func() {
+		var err error
```

Comment:
Why error declaration outside the `if` ?

Reply:
To avoid shadowing `pluginsTempDir`